create-issue/action.yml

```diff
@@ -0,0 +1,47 @@
+name: Create issue
+description: Create an issue in a repository
+author: ZhongRuoyu
+branding:
+  icon: alert-circle
+  color: blue
+inputs:
+  token:
+    description: GitHub token
+    required: false
+    default: ${{ github.token }}
+  repository:
+    description: Repository to create the issue in
+    required: false
+    default: ${{ github.repository }}
+  title:
+    description: The title of the issue
+    required: true
+  body:
+    description: The body of the issue
+    required: true
+  labels:
+    description: Comma-separated list of labels to add to the issue
+    required: false
+  assignees:
+    description: Comma-separated list of users to assign the issue to
+    required: false
+  dependent-step-conclusion:
+    description: >
+      Conclusion of the dependent step (`${{ steps.<step-id>.conclusion }}`);
+      used with `update-existing` and/or `close-existing`
+    required: false
+  update-existing:
+    description: >
+      Whether to post `body` as a comment under the issue with the same title,
+      if such an issue is found and `dependent-step-conclusion` is `failure`
+    required: false
+    default: "false"
+  close-existing:
+    description: >
+      Whether to close an existing issue with the same title as completed,
+      if such an issue is found and `dependent-step-conclusion` is `success`
+    required: false
+    default: "false"
+runs:
```

Comment:
I forgot to do this for `create-pull-request`, but you can add `outputs` here too: https://github.com/Homebrew/actions/blob/master/setup-homebrew/action.yml#L36.